scripts/lint.py

```diff
@@ -50,9 +50,6 @@ def lint_line(line: str) -> Optional[str]:
     if todo_pattern.search(line):
         return "TODO:s should be written as `TODO(yourname): what to do`"
 
-    if "rerurn" in line.lower():
-        return "Emil: you put an extra 'r' in 'Rerun' again!"
-
```

Comment:
have you tested it catches that? :D

Reply:
It warned about the `"rerurn"` part, so yes :)